catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -62,11 +82,10 @@ public String getWkt() throws CswException {
         }
 
         String crs = reader.getAttribute("crs");
-        // TODO: CRS value determines order of bounding box values, i.e.,
-        // WGS-84 specifies lon,lat order
-        // EPSG:4326 specifies lat,lon order
-        // For now we assume lat,lon order
-        LOGGER.debug("crs = {}", crs);
+        // For now, default an empty CRS to EPSG:4326
+        crs = StringUtils.isEmpty(crs) ? "urn:x-ogc:def:crs:EPSG:6.11:4326" : crs;
```

Comment:
any idea what the difference is between x-ogc and ogc?  The EPSG database just uses "ogc".  See http://www.epsg-registry.org/report.htm?type=selection&entity=urn:ogc:def:crs:EPSG::4326&reportDetail=long&style=urn:uuid:report-style:default-with-urn&style_name=OGP%20Default%20With%20Urn&title=